swagger/v1/move.json

```diff
@@ -74,8 +74,13 @@
             "readOnly": true
           },
           "date": {
-            "type": "string",
-            "format": "date",
+            "oneOf": [
```

Comment:
Isn't this equivalent to `"nullable": true`?

Reply:
https://swagger.io/docs/specification/data-models/data-types/#null

Reply:
Possibly, but I've kept this consistent with other data types

Reply:
I checked the existing API schema, and it seems we only use `nullable` for optional relationships. So I'd prefer to keep this consistent with everything else.